src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,92 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const peopleNames = people.map(({ name }) => name);
+  const { personSlug } = useParams();
+  const selectedPersonSlug = personSlug;
+
+  const findPersonByName = (personName: string) => {
+    return people.find(({ name }) => name === personName);
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+            slug,
+          } = person;
+
+          return (
+            <tr
+              data-cy="person"
+              key={slug}
+              className={classNames({
+                'has-background-warning': selectedPersonSlug === slug,
+              })}
+            >
+              <td>
+                <PersonLink person={person} />
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+
+              <td>
+                {motherName && peopleNames.includes(motherName)
```

Comment:
Better to prepare your people array after getting a response. Add `mother` and `father` to every person in `people` array